platforms/ios/demo/src/MapViewController.m

```diff
@@ -29,10 +29,9 @@ - (void)mapViewDidCompleteLoading:(TGMapViewController *)mapView
     // [mapView captureScreenshot:YES];
 }
 
-- (void)mapView:(TGMapViewController *)mapView didSceneReady:(int)sceneID Success:(bool)success WithError:(nonnull NSError *)sceneError
+- (void)mapView:(TGMapViewController *)mapView didSceneReady:(int)sceneID WithError:(nonnull NSError *)sceneError
```

Comment:
`WithError` doesn't match the code style of the public interface, can you update for `withError`?

`didLoadScene` might sound more natural regarding to objective-c guidelines.